Add text about compliance to Readme (#258)
timescale/timescale-prometheus: README.md

queries from their metrics data, and thus extract more meaningful
insights.

> ✅  We are the only long-term store for Prometheus data to have received a 

Author: ```suggestion
✅  Timescale-Prometheus is currently the only long-term store for Prometheus data to have received a 
```
